engine/src/flutter/tools/licenses/lib/patterns.dart

```diff
@@ -583,9 +557,9 @@ final List<RegExp> csReferencesByType = <RegExp>[
   // BoringSSL
   RegExp(
     kIndent +
-        r'Rights for redistribution and usage in source and binary forms are '
-                r'granted according to the (OpenSSL) license\. Warranty of any kind is '
-                r'disclaimed\.'
+        r'Implemented by Andy Polyakov <appro@fy\.chalmers\.se> for the (OpenSSL) project\. '
```

Comment:
Should we replace this pattern as
r'.* for the (OpenSLL) project\.'
so that we do not include the name and email-id here.

Reply:
This was based on an existing pattern for these sources that referenced the author:
https://github.com/flutter/flutter/pull/162913/files#diff-ece8a01f0de2f86b6219d7d8b49590355f9fd6c0f1059f0457c27c07d9bacae0L559